### PR TITLE
RHEL bump from 8.3 --> 8.4 in RNs (4.7 version)

### PR DESCRIPTION
**Pull request by bobfuru**
RHCOS got a bump from using RHEL 8.3 to RHEL 8.4. Currently, the OCP 4.7 RNs only point to 8.3. xref: http://mailman-int.corp.redhat.com/archives/coreos-devel/2021-August/msg00000.html

This PR updates this, and change management is also required.

Preview link:
https://deploy-preview-35261--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-7-release-notes?utm_source=github&utm_campaign=bot_dp#ocp-4-7-rhcos-rhel-8-3-packages

Relates to https://github.com/openshift/openshift-docs/pull/35265
Change management email sent Aug. 5

### CODE REVIEW

**Comment by bobfuru** [comment]
@travier and @miabbott PTAL for SME and QE review, please.

**Comment by miabbott** [comment]
LGTM

**Comment by travier** [comment]
LGTM as is but maybe we should mention the first .z stream release where we're making the switch?

**Comment by vikram-redhat** [comment]
LGTM

**Comment by sdodson** [comment]
Can you please include the detail as to which z-stream rebases to RHEL 8.4 in the release notes.

**Comment by travier** [comment]
4.7.24 _should_ include it (unreleased right now).

File: release_notes/ocp-4-7-release-notes.adoc
Comment (sdodson) [inline release_notes/ocp-4-7-release-notes.adoc:71]:
```suggestion
{op-system}  4.7.24 and above now use Red Hat Enterprise Linux (RHEL) 8.4 packages. {product-title} 4.6 and below will stay with RHEL 8.2 packages. This enables you to have the latest fixes, features, and enhancements, such as NetworkManager features, as well as the latest hardware support and driver updates.
```

Comment (bobfuru) [reply release_notes/ocp-4-7-release-notes.adoc]: Thanks, @sdodson! I incorporated your suggestion and am awaiting 4.7.24 GA to merge this content.

File: release_notes/ocp-4-7-release-notes.adoc
Comment (EricPonvelle) [inline release_notes/ocp-4-7-release-notes.adoc:71]: This is totally just a nitpick, but the phrase "OpenShift Container Platform 4.6 and below will stay with RHEL 8.2 packages." felt a little off to me, due to the verb "stay." My suggestion would be to rephrase it with another verb, but totally just a suggestion.

**Review by EricPonvelle — APPROVED** [review]
/lgtm

File: release_notes/ocp-4-7-release-notes.adoc
Comment (bobfuru) [reply release_notes/ocp-4-7-release-notes.adoc]: I agree, "stay" is confusing. I've updated based on your suggestion and to be in line with wording we are using in the update to 4.8 RNs (https://github.com/openshift/openshift-docs/pull/35265/files). Thanks!

**Comment by bobfuru** [comment]
Because the wording has changed a bit since the last round of approvals, @travier and @miabbott - could you PTAL again and ack if this wording looks good to merge once OCP 4.7.24 is GA?

**Review by travier — APPROVED** [review]
LGTM

**Comment by bobfuru** [comment]
[OCP 4.7.24](https://docs.openshift.com/container-platform/4.7/release_notes/ocp-4-7-release-notes.html#ocp-4-7-24) is GA now. With @miabbott on PTO, I plan to merge this today after peer review because it has been reviewed twice by Eng. and was already checked by QE previously.

**Pull request merged**
